frontend/src/Button/Button.module.css

```diff
@@ -49,11 +49,10 @@
 
 .icon {
   height: 64px;
-  padding: 16px;
 }
 
 .string {
-  padding: 12px 28px;
+  padding: 0px 16px;
```

Comment:
⚠️ action required: not in line with the new button design in figma, please change back to `12px 28px` (my bad for not fixing in my commits)